plugins/by-name/blink/default.nix

```
@@ -0,0 +1,71 @@
+{ lib, ... }:
+let
+  inherit (lib.nixvim) defaultNullOpts;
```

```suggestion
  inherit (lib) types;
  inherit (lib.nixvim) defaultNullOpts;
```

Your getting `error: undefined variable 'types'` in CI